assets/src/edit-story/components/panels/videoOptions.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+/**
+ * External dependencies
+ */
+import { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Switch } from '../form';
+import { SimplePanel } from './panel';
+import getCommonValue from './utils/getCommonValue';
+
+function VideoOptionsPanel({ selectedElements, onSetProperties }) {
+  const loop = getCommonValue(selectedElements, 'loop');
+  const [state, setState] = useState({ loop });
+  useEffect(() => {
+    setState({ loop });
+  }, [loop]);
+
+  const handleChange = (value) => {
+    const newState = {
+      loop: value,
+    };
+    onSetProperties(newState);
+    setState({ ...state, ...newState });
+  };
+
+  return (
+    <SimplePanel
+      name="videoOptions"
+      title={__('Video settings', 'web-stories')}
+    >
+      <Switch
```

Comment:
Needs design, but I am wondering whether a simple toggle/checkbox wouldn't suffice here.

Reply:
As discussed in last meeting. We know there is no design. So I just reused something that is already in. I think at this late stage, we shouldn't be building new things. We can loop back on this when we do have design.

Reply:
I am not suggesting building new things. We already have a toggle component.

Reply:
>I am not suggesting building new things. We already have a toggle component. 

I tried that but the toggle is designed for icons. A loop icon really doesn't clear here. It really need what that icon does.

Reply:
Argh, I meant the switch component: https://google.github.io/web-stories-wp/?path=/story/components-switch--default

But anyway, needs design first